src/ng/directive/select.js

```diff
@@ -4,9 +4,16 @@
 
 var noopNgModelController = { $setViewValue: noop, $render: noop };
 
-function setOptionAsSelected(optionEl) {
-  optionEl.prop('selected', true); // needed for IE
-  optionEl.attr('selected', true);
+function setOptionSelectedStatus(optionEl, value) {
+  optionEl.prop('selected', value); // needed for IE
+  if (value) {
+    optionEl.attr('selected', value);
```

Comment:
FWIW: `.attr('selected', null)` also removes the attribute. So, this could be:

```js
optionEl.prop('selected', value);
optionEl.attr('selected', value || null);
```